### ISSUE TITLE
Increase strictness / apply formatting

### ISSUE THREAD

**Issue by holtkamp** [dpovshed/octopus#2]
He @dpovshed FYI, since I really like your project and intend to use it / extend it, I created a fork. Development will be in the `develop` branch. https://github.com/holtkamp/octopus/tree/develop In case you are open for PRs, let me know, otherwise I will not disturb you anymore 😄

**Comment by dpovshed** [comment]
Hi Menno,

I am very glad to hear from you! I would love if you give Octopus a fresh boost!

I wrote Octopus for 2 goals: to solve a practical task - have a took for quick check of large sitemap; and to take a look at the library itself.

I liked ReactPHP very much; hope now it is even more stable - I did not worked with it during last 2 years.

**Comment by holtkamp** [comment]
done! https://github.com/dpovshed/octopus/releases/tag/0.1.0

**Comment by dpovshed** [comment]
Hi Menno,

I downloaded renewed Octopus, and it works nicely - I tested only standalone run. Nice to see that it is much more compliant with modern approach.

I just submitted a minor fix right into the master - related to documentation so the person who see it for the first time able to run it flawlessly.

Also I noted that currently we have only one option supported - _concurrency_. I believe that other options really give flexibility to the project. Especially, ability to choose get/head type of request and ability to save all the data (which is actually turns Octopus in a kind of simple Grabber).

I will create a separate issue to cover this. For sure, probably some options are unnecessary and some other could be added - how it was with UserAgent. Your participation is encouraged!
